docker/docker-compose.mysql-ssl-server-ssl.yml

```diff
@@ -18,13 +18,13 @@ services:
             # Mount the whole ./.acrakeys directory to be able generate keys and
             # place them in services' subdirectories
             - ./.acrakeys:/keys
-        # Please specify ACRA_CLIENT_ID environment variable, otherwise run with
-        # default 'testclientid' client id
+            - ./ssl/acra-client:/ssl:ro
         command: >-
-            --client_id=${ACRA_CLIENT_ID:-testclientid}
+            --client_id=""
+            --tls_cert=/ssl/acra-client.crt
             --generate_acrawriter_keys
             --keys_output_dir=/keys/acra-server
-            --keys_public_output_dir=/keys/acra-writer
+            --keys_public_output_dir=/keys/acra-server
```

Comment:
we can remove this line and leave only `keys_output_dir`. and in other files too

Reply:
Got it, let me fix it

Reply:
Discussed, to stay as is because of default behavior